vpc/service/assign_addresses_v3.go

```diff
@@ -607,6 +608,26 @@ func (vpcService *vpcService) AssignIPV3(ctx context.Context, req *vpcapi.Assign
 	return vpcService.assignIPsToENI(ctx, req, ass, maxIPAddresses, instance, trunkENI)
 }
 
+func addJumboRoute(ctx context.Context, req *vpcapi.AssignIPRequestV3, ass *vpcapi.AssignIPResponseV3) {
```

Comment:
how about just `addDefaultRoute` ? (the name of the function implies that it always does jumbo

Reply:
Done